src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/OffenderAlert.kt

```diff
@@ -98,6 +98,9 @@ class OffenderAlert(
   @OneToMany(mappedBy = "alert", fetch = LAZY, cascade = [CascadeType.ALL])
   var workFlows: MutableList<AlertWorkFlow> = mutableListOf(),
 
+  @Column(name = "CREATE_USER_ID")
+  var createUsername: String,
+
```

Comment:
first time we are propagating the create username  for sync, not sure if we need to  ...